### DISCUSSION TITLE
OSS License compatibility question

### DISCUSSION THREAD

**Discussion by Ashley123456789** [houtianze/bypy#636] [Q&A] [answered]
There’s some possible confusion on the license of your repository when you combine other open-source code. 

The modules `bypy/gui.py` and `bypy/__init__.py` claim their licenses as the **GPLv3 License**, a kind of viral license asking the derivative works to use GPL licenses as well. 
However, the license of your whole project is shown as the **MIT license** in LICENSE, i.e., **less strict** than the GPLv3 License on license terms, which has impacted the whole license compatibility in your repository and may bring legal and financial risks. Also, it may violate GPL regulations.

You can select another proper license for your repository or write exceptions for specific files.

**Comment by houtianze** [comment, marked as answer]
Thanks for the feedback, just updated the package to remove the (long time invalid) GPL text residue.
